### PR TITLE
fix: allow disabling group votes fetcher independently of epoch block fetcher

### LINKED ISSUE #10712: Celo: RPC node timeouts when fetching the epoch
1. Getting RPC node timeouts while fetching the epoch.
2. Constraint violations when inserting data to `celo_validator_group_votes` table.

### PR DESCRIPTION
**Pull request by fedor-ivn**
## Motivation

Closes #10712.

**:warning:  Note:** This PR should be merged after #10629.

### Enhancements

Added the `INDEXER_DISABLE_CELO_VALIDATOR_GROUP_VOTES_FETCHER` environment variable to allow more granular control over fetchers. Previously, disabling `INDEXER_DISABLE_CELO_EPOCH_FETCHER` also disabled the validator group votes fetcher. This new variable enables you to independently fetch validator group votes before indexing epochs, which helps avoid timeouts in networks with large block heights.

PR in docs: https://github.com/blockscout/docs/pull/322

### Bug Fixes

Corrected the primary key for the `celo_validator_group_votes` table by including `account_address_hash` and `group_address_hash`. This resolves the constraint violations caused by the previous omission of these fields from the primary key.

## Checklist for your Pull Request (PR)

- [ ] If I added new functionality, I added tests covering it.
- [ ] If I fixed a bug, I added a regression test to prevent the bug from silently reappearing again.
- [x] I checked whether I should update the docs and did so by submitting a PR to [docs repository](https://github.com/blockscout/docs).
- [x] If I added/changed/removed ENV var, I submitted a PR to [docs repository](https://github.com/blockscout/docs) to update the list of [env vars](https://github.com/blockscout/docs/blob/master/for-developers/information-and-settings/env-variables.md) and I updated the version to `master` in the Version column. If I removed variable, I added it to [Deprecated ENV Variables](https://github.com/blockscout/docs/blob/master/for-developers/information-and-settings/env-variables/deprecated-env-variables/README.md) page. After merging docs PR, changes will be reflected in these [pages](https://docs.blockscout.com/for-developers/information-and-settings/env-variables).
- [ ] If I added new DB indices, I checked, that they are not redundant, with PGHero or other tools.
- [ ] If I added/removed chain type, I modified the Github CI matrix and PR labels accordingly.

### CODE REVIEW

File: config/runtime.exs [hunk 1]
```diff
@@ -1037,12 +1037,15 @@ config :indexer, Indexer.Fetcher.PolygonZkevm.TransactionBatch.Supervisor,
 config :indexer, Indexer.Fetcher.Celo.ValidatorGroupVotes,
   batch_size: ConfigHelper.parse_integer_env_var("INDEXER_CELO_VALIDATOR_GROUP_VOTES_BATCH_SIZE", 200_000)
 
+config :indexer, Indexer.Fetcher.Celo.ValidatorGroupVotes.Supervisor,
+  enabled:
+    ConfigHelper.chain_type() == :celo and
+      not ConfigHelper.parse_bool_env_var("INDEXER_DISABLE_CELO_VALIDATOR_GROUP_VOTES_FETCHER")
```
Comment (vbaranov) [inline config/runtime.exs:1043]: Please add new var to `common-blockscout.env`

Comment (vbaranov) [reply config/runtime.exs:1043]: Also, is this intentional, that this fetcher is enabled by default?

Comment (fedor-ivn) [reply config/runtime.exs:1043]: Sure, this is intentional. Epoch fetcher is dependent on Validator group votes fetcher.

Comment (fedor-ivn) [reply config/runtime.exs:1043]: > Please add new var to common-blockscout.env

Processed in https://github.com/blockscout/blockscout/pull/10673/commits/0fed26811becf72ab8c947d23b0b1777965b8eda

**Review by vbaranov — APPROVED** [review]

**Pull request merged**